app/util/CyaTradingHistorySupport.scala

```diff
@@ -248,54 +248,96 @@ class CyaTradingHistorySupport @Inject() (
-                 Seq(
-                   SummaryListRow(
-                     key = Key(Text(messages("checkYourAnswersAboutTheTradingHistory.financialYearEndUpdates"))),
-                     value = Value(
-                       HtmlContent(
-                         financialYearEndDatesTable(
-                           financialYearEndDates = financialYearEndDates(sectionAnswers.answers.get),
-                           dateUtil = dateUtil
-                         ).body
+    if (forType == FOR6048) {
+      Html(
+        govukSummaryList(
+          SummaryList(rows =
+            sectionAnswers1
+              .conditionRow(
+                _.areYouVATRegistered.map(_.name).isDefined,
+                "checkYourAnswersAboutTheTradingHistory.areYouVATRegistered",
+                _.areYouVATRegistered.flatMap(_.name.capitalize),
+                controllers.aboutthetradinghistory.routes.AreYouVATRegisteredController.show,
+                "areYouVatRegistered"
+              )
+              .map(_.copy(classes = "no-border-bottom"))
+              ++
+                answerOpt(
+                  valueLabelKey = "checkYourAnswersAboutTheTradingHistory.financialYearEnd",
+                  valueOpt = sectionAnswers.answers
+                    .flatMap(_.occupationAndAccountingInformation)
+                    .flatMap(_.currentFinancialYearEnd.map(_.toMonthDay))
+                    .map(dateUtil.formatMonthDay),
+                  classes = if (!yearEndChanged) "" else "no-border-bottom"
+                ) ++
+                (if (yearEndChanged)
+                   Seq(
+                     SummaryListRow(
+                       key = Key(Text(messages("checkYourAnswersAboutTheTradingHistory.financialYearEndUpdates"))),
+                       value = Value(
+                         HtmlContent(
+                           financialYearEndDatesTable(
+                             financialYearEndDates = financialYearEndDates(sectionAnswers.answers.get),
+                             dateUtil = dateUtil
+                           ).body
+                         )
                        )
                      )
                    )
-                 )
-               else Seq.empty)
-        )
-      ).body
-    )
+                 else Seq.empty)
+          )
+        ).body
+      )
+    } else {
+      Html(
+        govukSummaryList(
+          SummaryList(rows =
+            sectionAnswers1
+              .conditionRow(
+                _.areYouVATRegistered.map(_.name).isDefined,
+                "checkYourAnswersAboutTheTradingHistory.areYouVATRegistered",
+                _.areYouVATRegistered.flatMap(_.name.capitalize),
+                controllers.aboutthetradinghistory.routes.AreYouVATRegisteredController.show,
+                "areYouVatRegistered"
+              )
+              .map(_.copy(classes = "no-border-bottom"))
+              ++
+                sectionAnswers
+                  .row(
```

Comment:
Can we please add conditional only onthe  unique part "occupationDate row"

Reply:
will do!